_episodes/03-running-containers.md

```diff
@@ -0,0 +1,234 @@
+~~~
+$ docker image ls
+~~~
+{: .language-bash}
+
+If you've just
+installed Docker, you won't see any images listed.
+
+To get a copy of the `hello-world` Docker image from the internet, run this command:
+~~~
+$ docker pull hello-world
+~~~
+{: .language-bash}
+
+You should see output like this:
+~~~
+Using default tag: latest
+latest: Pulling from library/hello-world
+1b930d010525: Pull complete
+Digest: sha256:f9dfddf63636d84ef479d645ab5885156ae030f611a56f3a7ac7f2fdd86d7e4e
+Status: Downloaded newer image for hello-world:latest
+docker.io/library/hello-world:latest
+~~~
+{: .output}
+
+> ## DockerHub
+>
+> Where did the `hello-world` image come from? Technically, it came from the DockerHub
+> website, which is a place to share Docker images with other people. More on that
+> in a later episode.
+{: .callout}
+
+> ## Exercise: Check on Your Images
+>
+> What command would you use to see if the `hello-world` Docker image had downloaded
+> successfully and was on your computer?
+> Give it a try before checking the solution.
+>
+> > ## Solution
+> >
+> > To see if the `hello-world` image is now on your computer, run:
+> > ~~~
+> > $ docker image ls
+> > ~~~
+> > {: .language-bash}
+> {: .solution}
+{: .challenge}
+
+Note that the downloaded `hello-world` image is not in the folder where you are in the terminal! (Run
+`ls` by itself to check.) The image is not a file like our normal programs and files;
+Docker stores it in a specific location that isn't commonly accessed, so it's necessary
+to use the special `docker image` command to see what Docker images you have on your
+computer.
+
+## Running the `hello-world` container
+
+To create and run containers from named Docker images you use the `docker run` command. Try the following `docker run` invocation. Note that it does not matter what your current working directory is.
+
+~~~
+$ docker run hello-world
+~~~
+{: .language-bash}
+~~~
+Hello from Docker!
+This message shows that your installation appears to be working correctly.
+
+To generate this message, Docker took the following steps:
+ 1. The Docker client contacted the Docker daemon.
+ 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+    (amd64)
+ 3. The Docker daemon created a new container from that image which runs the
+    executable that produces the output you are currently reading.
+ 4. The Docker daemon streamed that output to the Docker client, which sent it
+    to your terminal.
+
+To try something more ambitious, you can run an Ubuntu container with:
+ $ docker run -it ubuntu bash
+
+Share images, automate workflows, and more with a free Docker ID:
+ https://hub.docker.com/
+
+For more examples and ideas, visit:
+ https://docs.docker.com/get-started/
+~~~
+{: .output}
+
+What just happened? When we use the `docker run` command, Docker does three things:
+
+| 1. Starts a Running Container | 2. Performs Default Action | 3. Shuts Down the Container
```

Comment:
Table is rendering strangely, at least on my local version.  
<img width="918" alt="Screen Shot 2020-04-02 at 4 32 22 PM" src="https://user-images.githubusercontent.com/5558419/78302126-d22f0200-74ff-11ea-982f-ab1208cd54cc.png">

Reply:
I thought so. 😂 

this is definitely a line ending problem. if I make it all one line, should be fixed.

Reply:
try again?

Reply:
yay!  It looks great!